x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_significant_events_view/add_significant_event_flyout/add_significant_event_flyout.tsx

```diff
@@ -94,7 +94,7 @@ export function AddSignificantEventFlyout({
   }, [data.dataViews, definition.stream.name]);
 
   const { onGenerateDescription: generateDescription, onSaveDescription: saveDescription } =
-    useStreamDescriptionApi({ definition, refreshDefinition });
+    useStreamDescriptionApi({ definition, refreshDefinition, aiFeatures });
```

Comment:
can we pass the AIFeatures from the parent as well here instead of calling the hook so we don't see the unnecessary loading when opening the flyout and only call the connectors API once on the initial render

https://github.com/user-attachments/assets/08dfd768-deb6-4d7f-ad50-907ee4f89971